app/views/posts/index.html.erb

```diff
@@ -1,5 +1,24 @@
 <h1>Display posts</h1>
-<% @posts.each do |posts| %>
-    <li><%= post.title %></li>
-    <li><%= post.comments %></li>
-    <% end %>
\ No newline at end of file
+
+<div>
+  <img src="<%= @user.photo %>" alt="User Photo">
+</div>
+<h1><%= @user.name %></h1>
+<p>Number of posts: <%= @user.posts_counter %></p> 
+
+<% @posts.each do |post| %>
+  <li><%= post.title %></li>
+  <li><%= post.text %></li>
+  <li>Comments: <%= post.comments_counter %></li>
+  <li>Likes: <%= post.likes_counter %></li>
+  <% if @comments.present? %>
+<h2>Comments:</h2>
+<% @comments.each do |comment| %>
+  <p><%= comment.text %> - <%= comment.author.name %></p>
+<% end %>
+<% else %>
```

Comment:
[REQUIRED] Kindy use the `post.recent_comments` method to render the 5 most recent comments

Reply:
Thanks for the highlight @fobadara